internal/pkg/cli/deploy/env.go

```diff
@@ -184,6 +190,7 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*deploy.Creat
 		ArtifactBucketARN:    s3.FormatARN(partition.ID(), resources.S3Bucket),
 		ArtifactBucketKeyARN: resources.KMSKeyARN,
 		Mft:                  in.Manifest,
+		ForceUpdate:          in.ForceNewUpdate,
```

Comment:
It's kinda awkward that this field is set for `GenerateCloudFormationTemplate` but not for `DeployEnvironment` where that field gets populated instead in `UpdateAndRenderEnvironment` 💭 

What do you think of moving the logic to outside of `UpdateAndRenderEnvironment` into `DeployEnvironment` so that we do all the initialization in a single pkg?

Reply:
done!